pkg/util/validation/separate_metrics.go

```diff
@@ -30,3 +30,17 @@ func GroupLabel(o *Overrides, userID string, timeseries []mimirpb.PreallocTimese
 
 	return ""
 }
+
+// AttributionLabel obtains the value of cost attribution label for tenant
+func AttributionValue(attribLab string, userID string, lbs []mimirpb.LabelAdapter) string {
+	if len(lbs) == 0 {
+		return ""
+	}
+
+	for _, label := range lbs {
+		if label.Name == attribLab {
+			return label.Value
+		}
+	}
+	return ""
+}
```

Comment:
Why does this method need `userID`? And why do we need this method? How is it different from a usual label value getter?

Reply:
removed the helper function.